pkg/templates/templates.go

```diff
@@ -28,6 +28,12 @@ var (
 	kubeAPIResourceList []*metav1.APIResourceList
 )
 
+// TemplateContext is a struct that is provided during template execution.
```

Comment:
ClusterID was added here.
how to set that value?  we need to determine if   we can support that ,  if processes that call this api  like policy propagator  have access to the value and  provide that info  to the processor ?

Reply:
I thought I understood that Randy required it as part of architecture call, so I haven't yet looked how to provide that, but it seems that we will need to. This is why it is here.

On the config-policy-controller side, this would just not be set.

Reply:
While propagator  already has the ClusterName available as part of placement Decision, its doesnt already have ClusterId.  So the hesistance to just put it in already without figuring out  the backend.

It isnt actually needed for the team's  usecase either , the   request was  more like  might as well add that also along with clustername if needed later on.


I guess  you can leave it in their  for now ... depending on if we end up adding clusterId  or not I  might change this later.

Reply:
Good point. I will take it out and if we find an easy way to add this, we can add this back in later.